src/locales/en/translation.json

```diff
@@ -1,5 +1,11 @@
 {
   "welcome_message": "Welcome to Cogito's Ecommerce app API",
+  "disactvated": "your status has been disactivated",
+  "fail": "user not found",
+  "failId": "no user found with that ID",
+  "retrievedAll": "all users retrieved",
+  "retrieved": "user retrieved",
+  "updateStatus": "status updated successfully",
```

Comment:
- use snake case instead and remove fail because fail means a different thing. I would suggest casting the key to what the message says
- for fail you can reuse "user_not_found"
- here is an example of what you trying to do which is correct "user_not_found": "User not found",